CHANGELOG.MD

```diff
@@ -1,5 +1,35 @@
 ﻿# CHANGELOG
 
+## [1.19.1](https://github.com/PowerShell/PSScriptAnalyzer/tree/1.19.1)
+
+### Fixes
+
+- `UseCorrectCasing`: Do not use CommandInfoCache when CommandInfoParameters property throws due to runspace affinity problem of PowerShell engine (#1523) (by @bergmeister)
+- `ReviewUnusedParameter`: Do not trigger when MyInvocation.BoundParameters or PSCmdlet.MyInvocation.BoundParameters is used (#1520) (Thanks @jegannathanmaniganadan!)
+- `PipelineIndentationStyle.None`: Fix bug that caused incorrect formatting in hashtables (#1497) (by @bergmeister)
+- `UseUsingScopeModifierInNewRunspaces`: Fix ArgumentException when the same variable name is used in 2 different sessions. (#1493) (by @bergmeister)
+- `UseConsistentWhitespace`
+  - Check previous token only if it starts on the same line (#1491) (by @bergmeister)
+  - Fix CheckParameter bug when using interpolated string (#1498) (by @bergmeister)
+
+### New Rule
+
+- New rule (disabled by default): `AvoidUsingDoubleQuotesForConstantString` (#1470) (by @bergmeister)
```

Comment:
I know this probably differs from your established changelog convention, but this could maybe be a link to the documentation on the rule so that anyone reading the changelog can instantly write settings for it

Reply:
Yes good point. A link on the rule name is probably the best.

My changelog procedure is fairly minimal where I just dump the commit messages, append the author, group them manually and sometimes edit the text a little bit. That's also why I usually write a blog post as I'm aware the changelog is not the easiest to consume.

Reply:
Link now added